content/en/docs/troubleshooting/ClusterResourcePlacementEviction.md

```diff
@@ -0,0 +1,260 @@
+# Placement Eviction & Placement Disruption Budget Troubleshooting Guide
+
+This guide provides troubleshooting steps for issues related to placement eviction.
+
+An eviction object when created is ideally only reconciled once and reaches a terminal state. List of terminal states 
+for eviction are:
+- Eviction is Invalid
+- Eviction is Valid, Eviction failed to Execute
+- Eviction is Valid, Eviction executed successfully
+
+> **Note:** If an eviction object doesn't reach a terminal state, it is likely due to a failure in the reconciliation
```

Comment:
Hi Arvind! Just a nit, maybe it's better to move the note elsewhere or even drop it, considering that there are valid=false cases.